ConnectorRevit/ConnectorRevit/UI/ConnectorBindingsRevit.Settings.cs

```diff
@@ -109,10 +109,10 @@ public override List<ISetting> GetSettings()
         new MappingSetting
         {
           Slug = "receive-mappings",
-          Name = "Custom Type Mapping",
+          Name = "Missing Type Mapping",
           Icon = "LocationSearching",
           Values = mappingOptions,
-          Description = "Determine how incoming object types are mapped to object types in the host application"
+          Description = "Determines when the missing types dialog is shown\n\nNever: the dialog is never shown\nAlways: the dialog is always shown when types are missing\nFor New Types: the dialog is only shown if there are new unmapped types"
```

Comment:
Always will ALWAYS show the dialog. Even if there aren't any missing types. 

Use case for doing this is that there aren't any missing types incoming, but someone want to tryout a different floor so they map an existing type to a different existing type. Another reason for always showing it on always is because this is a new feature and there may be edge cases in which the missing incoming types counter will not work 100% correctly.

We could change this to align with what you've described if you don't find the use case compelling.

Reply:
Will fix this